interface/asset-manager.md

```diff
@@ -0,0 +1,42 @@
+# Asset Manager
+
+## Overview 
+The **Asset Manager** is an organized collection of **Assets** that can be readily created, imported, deleted and updated. Below, you will find a depiction of the **Asset Manager** and its features.
+
+![](../.gitbook/assets/interface/asset-manager/asset-manager.PNG) 
+
+## Views in the Assert Manager 
+The **Assert Manager** can be *toggled* between a `flat` or `folder` `tree` `view` by clicking on the *stripe icon* ![](../.gitbook/assets/interface/asset-manager/stripeIcon.PNG) located in the toolbar of the **Asset Manager**.
+
+The `flat tree` `view` shows **Assets** in a single-panel window while the `folder tree view` shows **Assets** in a dual-panel window. 
+
+The left panel of the `folder tree view` contains folders created by a user in order to separate different categories of **Assets**. If a folder is selected in the left panel, its corresponding contents will be displayed in the right panel.
+
+### Lock Current View
+A folder selected in the left panel of the `folder tree view` can be locked by clicking on the ![](../.gitbook/assets/interface/asset-manager/lockIcon.PNG) icon in the toolbar. Therefore, *clicking* on another folder in the left panel will have no effect in the right panel.
+
+## Import Assets
+New **Assets** can be imported in the following ways:
+
+* *Clicking* on the *plus icon* button ![](../.gitbook/assets/interface/asset-manager/plusIcon.PNG) in the toolbar and then the `Import Assets` button from the drop-down list.
+
+* *Right-clicking* in the left panel of the `folder tree view` > `Import Assets`. 
+
```

Comment:
Also dragging and dropping or using the Windows File Explorer (which is actually better because you can add folders then).